api/Dockerfile.dev

```
@@ -0,0 +1,41 @@
+
+# Development-only Dockerfile for the Rails API.
+# The production image lives in api/Dockerfile (Kamal / Thruster based).
+# This file is used exclusively by docker-compose.yml.
+
+ARG RUBY_VERSION=3.4.8
+FROM docker.io/library/ruby:${RUBY_VERSION}-slim
+
+WORKDIR /rails
+
+# Install system dependencies required to build native gems and run Rails.
+RUN apt-get update -qq && \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)